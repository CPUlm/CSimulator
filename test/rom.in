4
101
110
010
011
01
11
00
10
