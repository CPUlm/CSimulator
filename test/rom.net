INPUT i
OUTPUT o
VAR
  i:2, o:3
IN
o = ROM 2 3 i
