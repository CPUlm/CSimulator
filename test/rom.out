Step 1:
=> o = 010
Step 2:
=> o = 011
Step 3:
=> o = 101
Step 4:
=> o = 110
